crates/mir/src/db/queries/structs.rs

```diff
@@ -11,7 +11,7 @@ pub fn mir_lower_struct_all_functions(
     struct_
         .all_functions(db.upcast())
         .iter()
-        .map(|func| db.mir_lowered_func_signature(*func))
+        .map(|func| db.mir_lowered_pseudo_monomorphized_func_signature(*func))
```

Comment:
It seems weird to use the function because it is annotated as `NOTE: THIS SHOULD ONLY BE USED IN TEST CODE`.

Reply:
Ah, yes...I guess I forgot that this is also used by the graphviz stuff. I guess what I meant to say is that it should not be used in the actual compilation pipeline. The reason it is needed at all is because the std lib test start failing as soon as the std lib included generic code because the std lib tests iterate over these functions without them being actually called which means we can not monomorphize the generic parameters. So in order to be able to iterate over the functions I created a function that would populate them in a way that maps each generic param to `()`. There's probably a better way that I'm not seeing :)